docs/content/preview/yugabyte-voyager/release-notes.md

```diff
@@ -13,6 +13,42 @@ type: docs
 
 What follows are the release notes for the YugabyteDB Voyager v1 release series. Content will be added as new notable features and changes are available in the patch releases of the YugabyteDB v1 series.
 
+## v1.6 - November 28, 2023
+
+### New Features
+
+- Live migration
+
+  - Support for [live migration](../migrate/live-migrate/) from Oracle databases (with the option of [fall-back](../migrate/live-fall-back/)) {{<badge/tp>}}, using which you can fall back to the original source database if an issue arises during live migration.
+
+  - Various commands that are used in live migration workflows (including fall-forward) have been modified. Refer to [live migration](../migrate/live-migrate/) documentation for more details.
+
+  - A new command `yb-voyager get data-migration-report` has been added to display table-wise statistics during and post live migration.
+
+- End migration
+
+A new command `yb-voyager end migration` has been added to complete migration by cleaning up metadata on all databases involved in migration, and backing up migration reports, schema, data, and log files.
+
+### Enhancements
+
+- Boolean arguments in yb-voyager commands have been standardized as string arguments for consistent CLI usage. For example, use `--send-diagnostics true` to specify the `--send-diagnostics` argument.
```

Comment:
Minor change here as well: We want to be more explict  - 

In all yb-voyager commands, there is no need of explicitly using `=` while setting boolean flags to false; a white-space would work (just like arguments of other types). As a side effect of this, now you cannot use boolean flag names without any value e.g. `use --send-diagnostics true` instead of just `--send-diagnostics`.
In all yb-voyager commands, the boolean values can now be specified as `true/false, yes/no, 1/0, t/f, y/n`